test_plumbum.py

```diff
@@ -50,7 +50,34 @@ def test_no_template(self, mock_exit):
         templ, ns, region, filter_by, token = plumbum.interpret_options(args)
         self.assertEqual(ns, None)
         self.assertEqual(region, plumbum.DEFAULT_REGION)
-        self.assertEqual(filter_by, 'instance-type=c3.large')
+        self.assertEqual(filter_by, {u'instance-type': u'c3.large'})
```

Comment:
Nit: `self.assertEqual(filter_by, {'instance-type': 'c3.large'})` should be sufficient enough.